wsd/COOLWSD.cpp

```diff
@@ -3551,7 +3555,6 @@ class PrisonerRequestDispatcher final : public WebSocketHandler
 #else
             pid_t pid = 100;
             std::string jailId = "jail";
-            LOG_ASSERT_MSG(socket->getInBuffer().empty(), "Unexpected data in prisoner socket");
```

Comment:
What was wrong with this assertion? We don't expect residue data in the socket, regardless of whether there was data that we consumed, or it was always empty. What am I missing?